src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    private static String[] colorsList = new String[]{"yellow", "purple", "orange",
+            "green", "green", "blue", "red", "pink"};
+
+    public String[] getColorsList() {
+        return colorsList;
+    }
+
+    public static String randomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```
don't use static methods - they are in general bad practice

Reply:
done